src/test/rustdoc/assoc-consts.rs

```diff
@@ -1,8 +1,8 @@
 pub trait Foo {
     // @has assoc_consts/trait.Foo.html '//*[@class="rust trait"]' \
-    //      'const FOO: usize;'
+    //      'const FOO: usize = 13usize;'
```

Comment:
Since we already display `: type`, is there a need to put `usize` after `13`?

Reply:
Of course there is no need and I'd love to get rid of that suffix. However, `13usize` is returned from the `clean::Constant::value` method [I am using](https://github.com/rust-lang/rust/blob/19e1cf1639cd9ab95b42030cc80801f1af4ca7c6/src/librustdoc/html/render/mod.rs#L809) to evaluate and print a constant. I thought I'd rather use existing infrastructure.

Oh, only now I see that the suffix printing logic originates from `rustdoc`: `value()` uses [`print_evaluated_const`](https://github.com/rust-lang/rust/blob/4e1927db3c399fa34dc71992bd5dbec09f945c3d/src/librustdoc/clean/utils.rs#L262) / [`format_integer_with_underscore_sep`](https://github.com/rust-lang/rust/blob/4e1927db3c399fa34dc71992bd5dbec09f945c3d/src/librustdoc/clean/utils.rs#L277). I thought it was defined outside of `rustdoc`. Should I modify `print_evaluated_const`? Or add a parameter to it and thread it through?

Reply:
I see. Then please add a FIXME on your code generating it instead. It's out of scope for this PR.